### PR TITLE
Issue #212. Nearby competitions refresh after changing location or date.

### PR DESCRIPTION
**Pull request by pedrosino**

### CODE REVIEW

File: WcaOnRails/app/views/competitions/edit.html.erb [hunk 1]
```diff
@@ -139,47 +155,20 @@
...
-          <% else %>
-            <% if !@competition.has_date? %>
-              <div class="alert alert-danger" role="alert">You have not assigned a date for this competition yet.</div>
-            <% end %>
-            <% if !@competition.has_location? %>
-              <div class="alert alert-danger" role="alert">You have not assigned a location for this competition yet.</div>
-            <% end %>
-          <% end %>
+          <div id="nearby-competitions">
+            <%= render "nearby_competitions" %>
```
Comment (jfly) [inline WcaOnRails/app/views/competitions/edit.html.erb]: It looks like the new `_nearby_competitions` partial isn't part of this commit?

Comment (pedrosino) [inline WcaOnRails/app/views/competitions/edit.html.erb]: Fixed here: https://github.com/cubing/worldcubeassociation.org/commit/a2915da802575c07f7037cc47b4d43cfae430734

**Comment by jfly** [comment]
LGTM! @timhabermass, could you take a quick look?

**Comment by timhabermaas** [comment]
LGTM as well!

**Pull request merged**